vcr/assets/NutsAuthorizationCredential.config.yaml

```diff
@@ -0,0 +1,18 @@
+concept: authorization
```

Comment:
no template?

Reply:
Nope, that means the raw credential is returned.